### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SsparKluo**

### CODE REVIEW

**Comment by DinethP** [comment]
Can you make sure the format is correct in the README file. The last 4 lines in contributors are not properly formatted.
You can adjust this by adding a space and then "Enter" after each line.
For example, 
1155xxxxxx <- here is a space
1155xxxxxx

**Comment by iPhoneL** [comment]
> Can you make sure the format is correct in the README file. The last 4 lines in contributors are not properly formatted.
> You can adjust this by adding a space and then "Enter" after each line.
> For example,
> 1155xxxxxx <- here is a space
> 1155xxxxxx

Also, these names in contributors are duplicated: 1155108651.md, 1155107885.md, 1155115789.md.

**Comment by SsparKluo** [comment]
> > Can you make sure the format is correct in the README file. The last 4 lines in contributors are not properly formatted.
> > You can adjust this by adding a space and then "Enter" after each line.
> > For example,
> > 1155xxxxxx <- here is a space
> > 1155xxxxxx
> 
> Also, these names in contributors are duplicated: 1155108651.md, 1155107885.md, 1155115789.md.

I have delete these lines, i guess there are some things wrong when i pull the master into my branch

**Comment by DinethP** [comment]
So, you can't make the changes?

**Comment by DinethP** [comment]
@SsparKluo Don't just delete the lines, because we need those SID's to be displayed. Try to follow the above steps to make them format properly.

File: README.md [hunk 1]
```diff
@@ -24,10 +26,9 @@
 {% include_relative _stu/1155115789.md %}  
 {% include_relative _stu/1155104019.md %}  
 {% include_relative _stu/1155115792.md %}
-{% include_relative _stu/1155108651.md %}
-{% include_relative _stu/1155107885.md %}
-{% include_relative _stu/1155115789.md %} 
-{% include_relative _stu/1155098386.md %}
```
Comment (iPhoneL) [inline README.md]: Should keep {% include_relative _stu/1155098386.md %}, since it's not duplicated

Comment (DinethP) [reply README.md]: @iPhoneL Can i accept these changes now?

**Pull request merged**